src/doc/src/reference/unstable.md

```diff
@@ -1215,6 +1215,30 @@ cargo check -Z unstable-options -Z check-cfg=features,names,values
 
 * RFC: [#2906](https://github.com/rust-lang/rfcs/blob/master/text/2906-cargo-workspace-deduplicate.md)
 * Tracking Issue: [#8415](https://github.com/rust-lang/cargo/issues/8415)
+* [Current Summary](https://github.com/rust-lang/cargo/issues/8415#issuecomment-1112618913)
+* [Example Port](https://github.com/clap-rs/clap/pull/3719)
+
+### Testing notes
+
+We are looking for people to test `workspace-inheritance` and provide feedback about it.
+
+We would like to know
+* If there were any pain points in porting your project
+* Any errors or bugs that you found in testing
+* Performance concerns
+* Gaps in documentation
+
+Anything that you find please let us know by filing an issue or commenting on
+the [tracking issue](https://github.com/rust-lang/cargo/issues/8415).
+
+To get started make sure you are running a nightly version of cargo. After that 
+place `cargo-features = ["workspace-inheritance"]` at the top of any `Cargo.toml` you
+plan to use this feature in. Once those two things are done you can create a 
+`[workspace.package]` and `[workspace.dependencies]` in your workspace `Cargo.toml`.
+From there move any duplicate dependencies or package keys into their respective workspace 
+tables, and change them to `{key}.workspace = true` in the member `Cargo.toml`. 
+An example port has been made [in this PR](https://github.com/clap-rs/clap/pull/3719) as
+a "real-life" guide.
```

Comment:
I feel like a numbered list will better help people who "already know" steps to jump right to where they are needing